spec/helpers/vacancies_helper_spec.rb

```diff
@@ -39,7 +39,12 @@
 
     it 'should include job_role for legacy listings' do
       allow(vacancy).to receive_message_chain(:job_roles, :any?).and_return(false)
-      expect(helper.new_sections(vacancy)).to include('job_role')
+      expect(helper.new_sections(vacancy)).to include('job_details')
+    end
+
+    it 'should include subjects for legacy listings' do
+      allow(helper).to receive(:missing_subjects?).with(vacancy).and_return(true)
+      expect(helper.new_sections(vacancy)).to include('job_details')
```

Comment:
This expectation doesn't seem to match the description, which mentions subjects?